docs/reference/manifest/launchevent.md

```diff
@@ -40,16 +40,17 @@ For more information, see [Version overrides in the manifest](../../develop/add-
 |:-----|:-----|:-----|
 |  **Type**  |  Yes  | Specifies a supported event type. For the set of supported types, see [Configure your Outlook add-in for event-based activation](../../outlook/autolaunch.md#supported-events). |
 |  **FunctionName**  |  Yes  | Specifies the name of the JavaScript function to handle the event specified in the `Type` attribute. |
-|  **SendMode** (preview) |  No  | Required for `OnMessageSend` and `OnAppointmentSend` events. Specifies the options available to the user if your add-in stops the item from being sent. For available options, refer to [Available SendMode options](#available-sendmode-options-preview). |
+|  **SendMode** (preview) |  No  | Used by `OnMessageSend` and `OnAppointmentSend` events. Specifies the options available to the user if your add-in stops an item from being sent or if the add-in is unable to connect to the server. If the **SendMode** property isn't included, the `SoftBlock` option is set by default. For available options, refer to [Available SendMode options](#available-sendmode-options-preview). |
 
 ## Available SendMode options (preview)
 
-When you include the `OnMessageSend` or `OnAppointmentSend` event in the manifest, you must also set the **SendMode** property. The following are the available options. Based on the conditions your add-in is looking for, the user is alerted if your add-in finds an issue in the item being sent.
+When you include the `OnMessageSend` or `OnAppointmentSend` event in the manifest, you should also set the **SendMode** property. If the **SendMode** property isn't included, the `SoftBlock` option is set by default. The following are the available options. Based on the conditions your add-in is looking for, the user is alerted if your add-in finds an issue in the item being sent.
 
 | SendMode option | Description |
 |---|---|
 |`PromptUser`|In the alert, the user can choose to **Send Anyway**, or address the issue then try to send the item again.|
-|`SoftBlock`|The user must fix the issue before trying to send the item again.|
+|`SoftBlock`|The user is alerted that the item they're sending doesn't meet the add-in conditions and the add-in will decide what to do. For example, the add-in can be set to simply alert the user about the issue or require the user to address the issue before sending the item again. If the add-in is unable to connect to the server when the add-in is activated, the item will always be sent.|
```

Comment:
```suggestion
|`SoftBlock`|The add-in decides what to do if the item being sent doesn't meet the add-in's conditions. The add-in can be set to simply alert the user about the issue then the user decides if to **Send Anyway** or address the issue. Otherwise, the add-in can be set to require the user to address the issue before they try to send the item again.|
```